article-store/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6.4-jessie
+
+RUN mkdir -p /var/www && \
+    chown www-data:www-data /var/www
+
+RUN pip install pipenv
+
+RUN mkdir -p /srv/app && \
+    chown www-data:www-data /srv/app
+
+USER www-data
+WORKDIR /srv/app
+
+COPY --chown=www-data:www-data Pipfile Pipfile.lock ./
+RUN pipenv install --deploy
+
+COPY --chown=www-data:www-data . ./
```

Comment:
Usually the pattern for webserver (and by extension application server) user is that they run as `www-data` and all files outside of `/var/something` (or `var/something`) are owned by someone else (here it could be just `root`). That way compromised (Python) code still can't modify the code of the application. Therefore I'd strip the `--chown` in this file and push `USER www-data` down to the `CMD`.

Reply:
Agree on this, but the other apps aren't perfect either (since this is just throwaway) so don't mind whether this is done or not.

Reply:
Agree, though happy to leave for the time being.